flair/models/biomedical_entity_linking.py

```diff
@@ -946,9 +960,16 @@ class BiomedicalEntityLinker:
     entity / concept to these mentions according to a knowledge base / dictionary.
     """
 
-    def __init__(self, candidate_generator: AbstractCandidateGenerator, preprocessor: AbstractEntityPreprocessor):
+    def __init__(
+        self,
+        candidate_generator: AbstractCandidateGenerator,
+        preprocessor: AbstractEntityPreprocessor,
+        entity_type: str,
+    ):
         self.preprocessor = preprocessor
         self.candidate_generator = candidate_generator
+        self.entity_type = entity_type
+        self.annotation_layer = ENTITY_TYPE_TO_ANNOTATION_LAYER[self.entity_type]
```

Comment:
Good idea. However, it should be also possible to specify the (or even better a list of) annotation_layer directly by the user (because e.g. the user has trained his / her own NER model with it's own typing). 

Moreover I recognized that the annotation layer is different when the user loads an single entity type model (e.g. Classifier.load("hunflair-disease")).

Reply:
See comment below